packages/js/src/workouts/tailwind-components/steps/social-profiles/social-field-array.js

```diff
@@ -0,0 +1,92 @@
+import { TrashIcon } from "@heroicons/react/outline";
+import { PlusIcon } from "@heroicons/react/solid";
+import { __ } from "@wordpress/i18n";
+import { PropTypes } from "prop-types";
+import { useCallback } from "@wordpress/element";
+
+/**
+ * Specialized button to remove a profile URL.
+ *
+ * @param {number} index    The index of the text field to be removed.
+ * @param {func}   onRemove The callback to be executed upon clicking.
+ *
+ * @returns {Component} The FieldArray component.
+ */
+const RemoveProfileButton = ( { index, onRemove, buttonId } ) => {
+	const onRemoveHandler = useCallback( () => {
+		onRemove( index );
+	} );
+	return (
+		<button
+			id={ buttonId }
+			onClick={ onRemoveHandler }
+			className="yst-ml-2 yst-p-3 yst-text-gray-500 yst-rounded-md hover:yst-text-gray-600 focus:yst-text-gray-600 focus:yst-outline-none focus:yst-ring-2 focus:yst-ring-indigo-500"
+		>
+			<span className="yst-sr-only">{ __( "Delete item", "admin-ui" ) }</span>
+			<TrashIcon className="yst-relative yst--top-0.5 yst-w-5 yst-h-5" />
+		</button> );
+};
+RemoveProfileButton.propTypes = {
+	index: PropTypes.number.isRequired,
+	onRemove: PropTypes.func.isRequired,
+	buttonId: PropTypes.string.isRequired,
+};
+
+/**
+ * The FieldArray component.
+ *
+ * @param {array} data The array data.
+ * @param {Node} addButtonChildren Children for the add item button.
+ *
+ * @returns {Component} The FieldArray component.
+ */
+const SocialFieldArray = ( { items, onAddProfile, onRemoveProfile, onChangeProfile, addButtonChildren, fieldType: Component } ) => {
+	return (
+		<div>
+			{ items.map( ( item, index ) => (
+				<div key={ `url-${ index }` } className="yst-flex">
+					<div className="yst-flex-grow">
+						<Component
+							label={ __( "Other URL", "wordpress-seo" ) }
```

Comment:
This is my opinion, but I find it a bit strange to have multiple inputs labeled "Other URL" .

Perhaps we can leave the labels empty and have a little header section Reading "Other URLs"  or "Other Social profiles" or something at the top of all inputs (not each one individually).

Did you discuss this with UX?